src/utilities/index.ts

```diff
@@ -60,5 +60,34 @@ export const FormatBody = (message: string) => {
 }
 
 export function convertTimeStamp(timeStamp: string) {
-  return format(new Date(Number(timeStamp) * 1000), 'dd MMM yyyy | hh:mm a')
+  return format(new Date(Number(timeStamp) * 1000), "dd MMM yyyy | hh:mm a");
+}
+
+export async function httpRequest(url: RequestInfo, options?: RequestInit) {
```

Comment:
shouldn't there be a response Type as well?
![Screen Shot 2022-06-09 at 8 47 25 AM](https://user-images.githubusercontent.com/6158624/172756891-fa240fbb-36a5-4fdf-90a5-aec419ab591b.png)